jerry-core/mem/mem-allocator.cpp

```diff
@@ -53,26 +53,7 @@ mem_finalize (bool is_show_mem_stats) /**< show heap memory stats
 
   if (is_show_mem_stats)
   {
-    mem_heap_print (false, false, true);
-
-#ifdef MEM_STATS
-    mem_pools_stats_t stats;
-    mem_pools_get_stats (&stats);
-
-    printf ("Pools stats:\n");
-    printf (" Chunk size: %zu\n"
-            "  Pools: %zu\n"
-            "  Allocated chunks: %zu\n"
-            "  Free chunks: %zu\n"
-            "  Peak pools: %zu\n"
-            "  Peak allocated chunks: %zu\n\n",
-            MEM_POOL_CHUNK_SIZE,
-            stats.pools_count,
-            stats.allocated_chunks,
-            stats.free_chunks,
-            stats.peak_pools_count,
-            stats.peak_allocated_chunks);
-#endif /* MEM_STATS */
+    mem_stats_print ();
```

Comment:
@akiss77, for me it seems better to put `mem_stats_print` under `#ifdef MEM_STATS`, as without the define the call is no-op.